### PR TITLE
including {Apalache,Typing.tla}, README.md, LICENSE, and NOTICE

### PR DESCRIPTION
**Pull request by konnov**
This PR addresses #424

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  * Microsoft Z3
+    - URL: https://github.com/Z3Prover/z3
+    - License: MIT License
+
+  * Scala
+    - URL: https://www.scala-lang.org/
+    - License: Apache 2.0
+
+  * TBA  
+
```
Comment (shonfeder) [inline NOTICE:33]: rm?

**Review by shonfeder — APPROVED** [review]
LGTM, tho there's one line that seems out of place to me.

File: NOTICE [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  
+  * Microsoft Z3
+    - URL: https://github.com/Z3Prover/z3
+    - License: MIT License
+
+  * Scala
+    - URL: https://www.scala-lang.org/
+    - License: Apache 2.0
+
+  * TBA
```
Comment (konnov) [inline NOTICE:32]:
```suggestion
  * more to be added soon...
```

File: NOTICE (hunk 1 above)
Comment (konnov) [reply NOTICE]: It is not done. We have an open issue about licenses.

**Pull request merged**